test/verify/check-system-info

```diff
@@ -645,10 +645,25 @@ fi
         self.login_and_go('/system')
 
         m.execute("systemctl disable insights-client.timer")
-        b.wait_text("#insights_text", "Not connected to Insights")
+        b.wait_text(".system-health-insights a", "Not connected to Insights")
 
+        # Insights enabled but no results
         m.execute("systemctl enable insights-client.timer")
-        b.wait_not_present("#insights_text")
+        b.wait_not_present(".system-health-insights")
+
+        # With results.  We fake the files that `insights-client
+        # --check-results` would write.  This is less than ideal from
+        # an integration testing point of view, but the only
+        # alternative that we have right now would be to let
+        # insights-client access a mocked version of the Insights API.
+        # And since insights-client just writes the API response
+        # verbatim to disk anyway, we don't gain much by actually
+        # running it.
+
+        m.write("/var/lib/insights/host-details.json", '{ "results": [ { "id": "123-nice-id"  } ] }')
+        m.write("/var/lib/insights/insights-details.json", '[ { "rule": { "total_risk": 3 } }, { "rule": { "total_risk": 2 } }, { "rule": { "total_risk": 2 } }, { "rule": { "total_risk": 1 } } ]')
```

Comment:
The next commit adds a mock insights server, but then doesn't seem to use it. could this test use the mock server right away, or is there more plumbing necessary?

Reply:
`mock-insights` is used in `testInsightsStatus`.